coffee-electron/electron/trayGenerator.js

```diff
@@ -0,0 +1,56 @@
+const { Tray, Menu } = require('electron');
```

Comment:
This whole file also ripped straight from this [article](https://blog.logrocket.com/building-a-menu-bar-application-with-electron-and-react/)